filip/clients/ngsi_v2/cb.py

```diff
@@ -880,18 +881,39 @@ def get_subscription_list(self,
             self.log_error(err=err, msg=msg)
             raise
 
-    def post_subscription(self, subscription: Subscription) -> str:
+    def post_subscription(self, subscription: Subscription,
+                          update: Optional[bool] = False) -> str:
```

Comment:
You don't need Optional here because you have the default value. Since you definetly check for update it is  not optional.

Reply:
Yes you are right, I wanted to use Optional as a Hint for the user, but that is not how it is correctly used.